openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -956,6 +965,7 @@ def from_smiles(
         hydrogens_are_explicit: bool = False,
         allow_undefined_stereo: bool = False,
         _cls=None,
+        name: str = "",
```

Comment:
ditto here and many other places in this file - a `name` kwarg is added but then never used.